low-level/sspl_ll_d

```diff
@@ -422,14 +418,25 @@ def _sensors_actuators_factory(sspl_threaded_modules, msgQlist, operating_system
 
     # Read in list of sensors
     sensors = []
-    for group in sspl_settings_configured_groups:
-        sensors.extend(SSPL_SETTINGS[group].get("SENSORS"))
+    sensors_list = Conf.get(SSPL_CONF, f'{SSPL_SETTING}>{SENSORS}', [])
+    always_enable_sensors = Conf.get(SSPL_CONF,
+        f'{SSPL_SETTING}>{ALWAYS_ENABLE_SENSORS}', [])
+    for sensor in sensors_list:
+        # to start sensor thread add sensor into sensors list only if it is
+        # present in alywas_enable_list or if it is present in
+        # sspl_configured_groups.
+        # (sensor name present in sspl_configured_groups
+        # only if 'monitor' field value is True)
+        if (sensor.startswith('RealStor') and
```

Comment:
Am not getting the need for this if-elif need. Also with my comment in config file, to avoid always_enable_sensor, we may not need this even, unless you can tell me some usecase

Reply:
'elif' is not needed, as  always_enable_sensor list is removed.
  
  for all realstor sensors we have one common sections defined in config file and in conf_based_sensors file i.e. 'REALSTORSESORS'. In that section 'monitor' key is present. 
  
   here we are checking 'if (sensor.startswith('RealStor')' then check 'monitor' field from REALSTORSENSORS section if it true then start sensor otherwise not.